tests/unit/libs/fetcher.js

```diff
@@ -243,7 +243,7 @@ describe('Server Fetcher', function () {
                     },
                     req = {
                         method: 'GET',
-                        path: '/resource/' + mockService.name + ';' + qs.stringify(params, ';')
+                        path:  mockService.name + ';' + qs.stringify(params, ';')
```

Comment:
@lenage this seems to be problem.

instead of removing `/resource/`, replace it with `/`. Same with the below test cases.